CODE_OF_CONDUCT.md

```diff
@@ -13,44 +41,43 @@ We are committed to providing a friendly, safe and welcoming environment for all
 
 ### Our Goal
 
 The goal of this document is to set the overall tone for our community. This isn’t an exhaustive list of things you can and can't do. Rather, take this document in the spirit in which it’s intended, and try to be your best self.
 
-We value many things beyond technical expertise, including collaboration and supporting others within our community. Providing a positive experience for other community members can have a much more significant impact than simply providing the correct answer.
-
 ### Scope
 
-This Code of Conduct applies to all spaces managed by NoBoilerplate Discord. This includes, but is not limited to, the Discord server, our repositories on GitHub, the YouTube-channel, and meet-ups. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+This Code of Conduct applies to all spaces managed by 0atman/Namta0/Tris. This includes, but is not limited to, the NAMTAO Productions Discord server, NoBoilerplate/LostTerminal/ModemPrometheus repositories on GitHub and the NoBoilerplate YouTube-channel. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
 
 The NoBoilerplate Code of Conduct applies equally to all members of the community, including staff.
 
 ---
 
-# Code of Conduct Policies
-
 ### Moderation Policy
 
-These are the policies for upholding our community’s rules and the code of conduct. If you want to report a situation that needs to be reviewed by our moderation team, please see our [reporting guide](https://www.pythondiscord.com/pages/code-of-conduct/#reporting-guide).
+These are the policies for upholding our community’s rules and the code of conduct. If you want to report a situation that needs to be reviewed by our moderation team, please direct-message the Modmail bot on the NAMTAO Productions Discord.
 
-1.  The [Python Discord Code of Conduct](https://www.pythondiscord.com/pages/code-of-conduct/#code-of-conduct) and the [Community Rules](https://www.pythondiscord.com/pages/rules) are enforced by the moderation team, which consists of users with the Moderators, Admins or Owners role on the NoBoilerplate Discord server.
-2.  Behavior that moderators find inappropriate, whether listed in the code of conduct, the community rules, or not, is also not allowed.
-3.  Complaints about moderation in-channel are not allowed. If a moderator takes an action or makes a decision you do not agree with, please send a Direct Message (DM) to ModMail from our Discord server.
-4.  If you disagree with a moderation action or decision taken against you, you may appeal the action or decision by following the [Appeal Procedure](https://www.pythondiscord.com/pages/code-of-conduct/#appeal-procedure).
+1.  The Code of Conduct and the Community Rules are enforced by the moderation team, which consists of users with the moderators role on the NoBoilerplate Discord server.
```

Comment:
maybe change the wording slightly because "with the moderators" confused me at first. maybe add backticks?